pkg/manager-nvme/manager.go

```diff
@@ -217,8 +217,9 @@ func findStoragePool(storageId, storagePoolId string) (*Storage, *interface{}) {
 	return nil, nil
 }
 
+// CleanupVolumes - remove all volumes other than the list of providingVolumes
 func CleanupVolumes(providingVolumes []ProvidingVolume) {
-	for _, storage := range mgr.storage {
+	for _, storage := range GetStorage() {
```

Comment:
This change causes the capacity that is deleted to be returned to the unallocated space.